python/tests/res/job_queue/test_jobmanager.py

```diff
@@ -368,7 +368,7 @@ def test_data_from_forward_model_json(self):
             self.assertEqual( "/path/to/data" , os.environ["DATA_ROOT"])
             self.assertEqual( "ERT_RUN_ID", os.environ["ERT_RUN_ID"])
 
-    def test_data_from_forward_model_json(self):
+    def test_data_from_forward_model_json_no_root(self):
```

Comment:
Renamed this test as it already was another test with the same name. Nothing to do with my changes.